client/src/panels/panel_declarations/services/provided_services_list.js

```diff
@@ -1,30 +1,80 @@
-import text from "./services.yaml";
+import text from "./provided_services_list.yaml";
 import { Service } from "../../../models/services.js";
+import { create_text_maker } from "../../../models/text.js";
+import { TM } from "../../../components/index.js";
 
-import {
-  create_text_maker_component,
-  InfographicPanel,
-  declare_panel,
-} from "../shared.js";
+import { InfographicPanel, declare_panel } from "../shared.js";
+import { FancyUL } from "../../../components";
 
-const { text_maker, TM } = create_text_maker_component(text);
+const text_maker = create_text_maker(text);
 
-const ProvidedServicesListPanel = ({ panel_args }) => (
-  <div>
-    <TM
-      k="list_of_provided_services_desc"
-      args={{
-        subject_name: panel_args.subject.name,
-        num_of_services: panel_args.services.length,
-      }}
-    />
-    {_.map(panel_args.services, ({ name, id, org_id }) => (
-      <a key={id} href={`#dept/${org_id}/service-panels/${id}`}>
-        {name}
-      </a>
-    ))}
-  </div>
-);
+class ProvidedServicesListPanel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { service_query: "" };
+  }
+
+  render() {
+    const { panel_args } = this.props;
+    const { service_query } = this.state;
+
+    return (
+      <div>
+        <TM
+          tmf={text_maker}
+          k="list_of_provided_services_desc"
+          args={{
+            subject_name: panel_args.subject.name,
+            num_of_services: panel_args.services.length,
+          }}
+        />
+        <input
+          aria-label={text_maker("explorer_search_is_optional")}
+          className="form-control input-lg"
+          type="text"
+          style={{ width: "100%", marginBottom: "1rem", marginTop: "2rem" }}
+          placeholder={text_maker("filter_results_service")}
+          onChange={(evt) => this.setState({ service_query: evt.target.value })}
+          value={service_query}
+        />
+        <FancyUL>
+          {_.map(
+            _.filter(
+              panel_args.services,
+              (service) =>
+                _.includes(
+                  service.name.toLowerCase(),
+                  service_query.toLowerCase()
+                ) ||
+                _.includes(
+                  service.service_type.toLowerCase(),
+                  service_query.toLowerCase()
+                )
+            ),
+            ({ name, id, org_id, service_type, description }) => (
+              <React.Fragment key={id}>
+                <a href={`#dept/${org_id}/service-panels/${id}`}>{name}</a>
+                <p>{description}</p>
+                <div
+                  style={{
+                    display: "flex",
+                    fontSize: "14px",
+                    justifyContent: "space-between",
+                  }}
+                >
+                  <div className="tag-badge btn-ib-primary">{service_type}</div>
```

Comment:
ah I actually meant `btn-ib-primary` style for `See the service`. My bad on not being too clear on that haha